src/Adapter/MSTest.TestAdapter/Execution/TestAssemblyInfo.cs

```diff
@@ -95,6 +95,11 @@ internal set
     /// </summary>
     public Exception? AssemblyInitializationException { get; internal set; }
 
+    /// <summary>
+    /// Gets or sets the assembly cleanup exception.
+    /// </summary>
+    internal Exception? AssemblyCleanupException { get; set; }
```

Comment:
As far as I can see, it's only set from this class so the setter should be `private`.

Reply:
Updated,